### DISCUSSION TITLE
Alternate method for CRL check in syslog-ng

### DISCUSSION THREAD

**Discussion by BRKiranAT** [syslog-ng/syslog-ng#5195] [Q&A] [answered]
Hello All,
 
Seeking for your support on below query.
 
In order to use the CRL check in the syslog-ng application, it is necessary to create a hashed link file for the CRL. The following commands need to be executed to generate the link file:
openssl crl -in crlfile.crl -noout -hash
ln -s crlfile.crl 6d2962a8.r0
 
Since this process can be somewhat complicated, is there any way to enable CRL checking without generating the hashed link file?
 
Thanks in advance!

**Comment by HofiOne** [comment, marked as answer]
Hi,
There is no other option currently, this is the intended way to do it.
https://syslog-ng.github.io/admin-guide/100_TLS-encrypted_message_transfer/004_TLS_options#crl-dir

**Reply by BRKiranAT** [reply]
Thank you for the info.
